docs/deliverables-portal.md

```diff
@@ -0,0 +1,134 @@
+# Deliverables Portal
+## Overview
+Sampuru is a system for monitoring the quality control ('QC') status of items in the Ontario Institute for Cancer Research (OICR) Labs. 
+
+The Deliverables portal will be a page in Sampuru where internal users are able to input/update deliverable information pertinent to a case. The proposed design for the portal is an editable table with the following columns:
```

Comment:
The future tense reads weird and we'd need to switch it to present tense when development is done anyway, so maybe it should be in the present tense from the start?